pages/phases/reviewing-resumes/prep.md

```diff
@@ -20,10 +20,20 @@ Before you send resumes to SMEs for review, mark applicants as ineligible if:
 
 Instead of reviewing resumes to determine minimum qualification, review the technical justification that SMEs provide to make sure their decision can be retraced later on. As HR specialist, you'll adjudicate veterans' preference after SMEs conduct phone assessment interviews.
 
-### Create Applicant Lists for SME Reviews
+### Decide on a Resume Review Platform for your SMEs
+
+1. Create accounts for your SMEs to log onto your talent aquisition system to conduct their review. Monster customers who piloted SME-QA have used the Collaborate add-on to have SMEs review resumes and leave notes for HR to review. We have instructions manuals available upon request. USA Staffing customers can create "hiring manger" roles for the SMEs and assign stored lists for a pair of SMEs to review.  SMEs can leave their feedback in the "notes" field and HR can create a third stored list for the tiebreaker. In this process, SMEs have to reference the required competencies/proficiencies in a separate file. 
+
+*Lessons learned from our pilots*: SMEs conducting resume review in USA Staffing found the 15-minute time outs after inactivity to be a significant challenge to completing this work.  They also struggled with remembering every requirement they had to review since their feedback was in a free-form notes field. HR had to send a lot of reviews back to SMEs to redo and clarify. Using USA Staffing for the resume review phase took took 19 and 12 days to complete in our first two pilots.
```

Comment:
```suggestion
*Lessons learned from our pilots*: SMEs struggled with USA Staffing timing them out after 15 minutes of inactivity.  They also struggled to remember every requirement for their review, since their feedback was in a free-form notes field. HR had to send a lot of reviews back to SMEs to redo and clarify. Using USA Staffing for the resume review phase took took 19 and 12 days to complete in our first two pilots.
```